dojo_toolkit/main.py

```diff
@@ -42,11 +47,11 @@ def on_modified(self, event):
         process = Popen([cmd], shell=True)
         process.wait()
         if process.returncode == 0:
-            notification.update('OK TO TALK', '', 'g.jpg')
+            notification.update('OK TO TALK', '', '')
```

Comment:
if not pass the image here and in below, some implementations of libnotify (such as twmn) does not show it.